adagrams/game.py

```diff
@@ -1,11 +1,162 @@
+#Maggie T adagrams project 
+# Billie Betts adagram project
+
+import string
+import random
+
+quantity_data = {1: ['Z', 'X', 'K', 'Q', 'J'],
+                  2: ['B','C','F','H','M', 'P','V','W','Y'],
+                  3: ['G'],
+                  4: ['D', 'L', 'S', 'U'],
+                  6: ['N', 'R', 'T'],
+                  8: ['O'],
+                  9: ['A', 'I'],
+                  12: ['E']}
+
+value_data = {1: ['A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T'],
+              2: ['D','G'],
+              3: ['B', 'C', 'M', 'P'],
+              4: ['F', 'H', 'V', 'W', 'Y'],
+              5: ['K'],
+              8: ['J', 'X'],
+              10: ['Q', 'Z']}
+
+#Takes the two inputs and stores them in one dictionary 
+def generate_alphabet_data_list(quantity_data, value_data):
+  all_letters = string.ascii_uppercase
+  results = [] #dictionaries stored in results
+  for letter in range(len(all_letters)):
+    generate_dict = {}
+    generate_dict['letter'] = all_letters[letter]
+
+    for key, value in quantity_data.items():
+      if all_letters[letter] in value:
+        generate_dict['quantity'] = key
+
+    for key, value in value_data.items():
+      if all_letters[letter] in value:
+        generate_dict['value'] = key
+  
+    results.append(generate_dict)
+
+  return results
+  
+
+alphabet_data_list = generate_alphabet_data_list(quantity_data, value_data)
+#makes a list that generates the numbers of letters so its proportionate to the probability of getting a certain letter
+def letter_soup(letter_data):
```

Comment:
You could simplify this program if you added a second argument to random.choices to select a letter that is weighted by the quantity amount.